docs/data/data-grid/server-side-data/tree-data.md

```diff
@@ -4,46 +4,55 @@ title: Data Grid - Server-side tree data
 
 # Data Grid - Server-side tree data [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
 
-<p class="description">Tree data lazy-loading with server-side data source.</p>
+<p class="description">Implement lazy-loading server-side tree data in the Data Grid using the Data Source layer.</p>
 
-To dynamically load tree data from the server, including lazy-loading of children, you must create a data source and pass the `dataSource` prop to the Data Grid, as detailed in the [overview section](/x/react-data-grid/server-side-data/).
+## Server-side tree data
+
+Trees are hierarchical data structures that organize data into parent-child relationships.
+The Data Grid Pro can handle server-side tree data to render grouped rows with nested children using the [Data Source layer](/x/react-data-grid/server-side-data/#the-solution-the-data-source-layer).
```

Comment:
I wonder if the icon is not enough to convey which plan a feature falls under. If not, should we repeat the plan information for the rest of the commercial features too?